### ISSUE TITLE
Disclosure - broken flexbox in safari 14 [NDS-93]

### ISSUE THREAD

**Issue by JRebella** [wwnorton/design-system#177]
![image](https://user-images.githubusercontent.com/47323512/127334138-5db66d9f-2a45-46e2-83bb-054785c04f0d.png)

Tracked this down to a bug in safari itself where some HTML elements can't be flex, in this case the `<summary>` is the issue: https://github.com/philipwalton/flexbugs#flexbug-9

> Workaround
> The simple solution to this problem is to use a wrapper element that can be a flex container (like a `<div>`) directly inside of the element that can't. Demos 9.1.b and 9.2.b show workaround for the `<button>` and `<fieldset>` elements, respectively.
[NDS-93]

[NDS-93]: https://wwnorton.atlassian.net/browse/NDS-93

**Comment by sh0ji** [comment]
Nice find, thanks for opening this! If I'm understanding, the fix would be something like this?

```html
<summary> <!-- currently this is a flex box -->
  <div style="display: flex;"> <!-- move flex to here -->
    ...
  </div>
</summary>
```

**Comment by JRebella** [comment]
> 
> 
> Nice find, thanks for opening this! If I'm understanding, the fix would be something like this?
> 
> ```
> <summary> 
>   <div style="display: flex;"> 
>     ...
>   </div>
> </summary>
> ```

Yup, just tested it with the console inspector and it seems to be working as expected:

![image](https://user-images.githubusercontent.com/47323512/127502805-0de03e72-2b70-4139-aa49-291bb024a79d.png)

**Comment by sh0ji** [comment]
Closed by #187.
